README.md

```diff
@@ -292,6 +292,83 @@ test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; fini
      Summary [   0.005s] 1 test run: 1 passed, 66 skipped
 ```
 
+## Use cases: augmenting static analysis/debuggers
+
+By default the `Facet` derive macro creates and exports
+a global static variable `{UPPER_CASE_NAME}_SHAPE` referencing the
+`Shape` of the derived `Facet` trait.
+
+Furthermore, `Shape` and all nested fields are `#[repr(C)]`.
+
+This information can be used by external processes (like debuggers) to access the
+layout and vtable data.
+
+For example, suppose we have:
+
+```rust,ignore
+#[derive(Debug, Facet)]
+struct TestStruct {
+    field: &'static str,
+}
+
+static STATIC_TEST_STRUCT: TestStruct = TestStruct {
+    field: "some field I would like to see",
+};
+```
+
+By default, printing this in `lldb` returns the lengthy:
+
+```bash
+(lldb) p STATIC_TEST_STRUCT
+(simple_test::TestStruct) {
+  field = "some field I would like to see" {
+    [0] = 's'
+    [1] = 'o'
+    [2] = 'm'
+    [3] = 'e'
+    [4] = ' '
+    [5] = 'f'
+    [6] = 'i'
+    [7] = 'e'
+    [8] = 'l'
+    [9] = 'd'
+
+  ... (and so on)
+}
+```
+
+However, the `TestStruct::SHAPE` constant is available at `TEST_STRUCT_SHAPE`:
```

Comment:
A small detail I elided for brevity: currently the symbol is still mangled and is actually something like:
```
__ZN11simple_test17TEST_STRUCT_SHAPE17h5e18cd1cf07f11deE
```

However, `lldb` is smart enough to figure this out, and I figure most people who would have this use case could also figure that out.